### DISCUSSION TITLE
Vendure Admin "Bearer Token" Config/Error

### DISCUSSION THREAD

**Discussion by ijkml** [vendurehq/vendure#592] [Q&A] [answered]
Setting `authOptions.tokenMethod` to 'bearer' works great on the shop-api (as long as you don't forget to add `{"authorization": "Bearer _token_"}` in the header), but how do you configure **VendureAdmin** to do that? Right now, I can't log in to the admin dashboard.

**Comment by michaelbromley** [comment, marked as answer]
Hi,

Yes this is not easily discoverable in the docs, but in your setup of the AdminUiPlugin you can specify the token method: https://www.vendure.io/docs/typescript-api/admin-ui-plugin/admin-ui-config/#tokenmethod

```ts
AdminUiPlugin.init({
  adminUiConfig: {
    tokenMethod: 'bearer',
  },
})
```
